.circleci/config.yml

```diff
@@ -85,8 +85,9 @@ aliases:
 
   - &make_test
     name: "Running unit tests"
+    # Load OpenQA::Test::PatchDeparse after Devel::Cover
     command: |
-      export PERL5OPT="$COVEROPT,-db,cover_db_${CIRCLE_JOB}"
+      export PERL5OPT="$COVEROPT,-db,cover_db_${CIRCLE_JOB} -It/lib -MOpenQA::Test::PatchDeparse"
```

Comment:
great that you found a way that does not repeating the include in every test file. However I think this should be placed within the Makefile as it's not specific to circle CI but cover in general, right?

Reply:
I'm fighting with make, trying to expand PERL5OPT in the test target...
Think I got it now

Reply:
done